src/openapi.yaml

```diff
@@ -6728,6 +6688,239 @@ components:
           type: string
           maxLength: 1000
           minLength: 1
+      discriminator:
+        propertyName: betrokkene_type
+    RolNatuurlijkPersoon:
+      title: betrokkene_identificatie
+      type: object
+      properties:
+        burgerservicenummer:
+          title: Burgerservicenummer
+          description: Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet
+            algemene bepalingen burgerservicenummer.
+          type: string
+          maxLength: 9
+        nummerAnderNatuurlijkPersoon:
+          title: Nummer ander natuurlijk persoon
+          description: Het door de gemeente uitgegeven unieke nummer voor een ANDER
+            NATUURLIJK PERSOON
+          type: string
+          maxLength: 17
+        aNummer:
+          title: A nummer
+          type: integer
```

Comment:
I *think*, given how it's used, this should rather be a string. Or, otherwise, the `minimum` value should be 0.

Reply:
`a_nummer` field is changed to CharField with Regex validator